tools/check/check-bazel-prepare.sh

```diff
@@ -19,6 +19,7 @@
 #  -o pipefail: sets the exit code of a pipeline to that of the rightmost command to exit with a non-zero status,
 #       or to zero if all commands of the pipeline exit successfully.
 set -euo pipefail
+rm -rf /home/jenkins/.cache/bazel/_bazel_jenkins/install/a09dbb90c658248f08f9aa0eba11997d
```

Comment:
Could you explain this change?

Reply:
bazel building system leaves an invalid cache in our CI, need to manually delete it. Will send you details in our IM